File: libgomp/testsuite/libgomp.fortran/omp-control-tools-1.f90
       use omp_lib
       implicit none
       !include "omp_lib.h"
       if (omp_control_tool(omp_control_tool_start, 1)                          &
     &    /= omp_control_tool_notool) stop 1
       if (omp_control_tool_start /= 1                                          &
     &    .or. omp_control_tool_pause /= 2                                      &
     &    .or. omp_control_tool_flush /= 3                                      &
     &    .or. omp_control_tool_end /= 4                                        &
     &    .or. omp_control_tool_max /= 2147483647                               & ! INT32_MAX
     &    .or. omp_control_tool_notool /= -2                                    &
     &    .or. omp_control_tool_nocallback /= -1                                &
     &    .or. omp_control_tool_success /= 0                                    &
     &    .or. omp_control_tool_ignored /= 1) stop 2
       if (kind(omp_control_tool_result_kind) /= 4                              &
     &    .or. omp_control_tool_kind /= 4) stop 3
       end

File: libgomp/testsuite/libgomp.c/omp-control-tools-1.c
#include <omp.h>

int
main ()
{
  omp_control_tool_t cmd = omp_control_tool_start;
  omp_control_tool_result_t res;
  res = omp_control_tool (omp_control_tool_start, 1, nullptr);
  if (res != omp_control_tool_notool)
    __builtin_abort ();
  if (omp_control_tool_start != 1
      || omp_control_tool_pause != 2
      || omp_control_tool_flush != 3
      || omp_control_tool_end != 4
      || omp_control_tool_max != 2147483647 /* INT32_MAX */
      || omp_control_tool_notool != -2
      || omp_control_tool_nocallback != -1
      || omp_control_tool_success != 0
      || omp_control_tool_ignored != 1)
    __builtin_abort ();
}
